turbinia/processors/mount_local.py

```diff
@@ -212,7 +226,26 @@ def PreprocessLosetup(
           losetup_command, universal_newlines=True).strip()
     except subprocess.CalledProcessError as e:
       raise TurbiniaException('Could not set losetup devices {0!s}'.format(e))
-
+    # Make sure we have a proper loop device.
+    for _ in range(RETRY_MAX):
+      if IsLosetup(losetup_device):
+        log.info(
+            'Loop device {0:s} successfully attached'.format(losetup_device))
+        break
+      if os.path.exists(losetup_device):
+        log.info(
+            'Loop device {0:s} mode is {1}'.format(
+                losetup_device,
+                os.stat(losetup_device).st_mode))
+      time.sleep(1)
+    message = None
+    if not os.path.exists(losetup_device):
+      message = 'Loop device path {0:s} does not exist'.format(losetup_device)
+    elif not IsLosetup(losetup_device):
+      message = 'Device {0:s} is not a loop device'.format(losetup_device)
```

Comment:
Maybe just to be clear this could read something like "is not attached according to losetup"